data/data/azure/cluster/vnet/internal-lb.tf

```diff
@@ -0,0 +1,18 @@
+data "azurerm_lb" "internal" {
+  name                = "${var.cluster_id}-internal"
+  resource_group_name = var.resource_group_name
+}
+
+data "azurerm_lb_backend_address_pool" "internal_lb_controlplane_pool_v4" {
```

Comment:
Is this data source used? I couldn't find it being used and it looks like the availability set for masters is based on the `ilb_backend_pool_v4_id` variable from the vnet output.

Reply:
I've nuked the cluster/vnet directory. You were right in that it wasn't necessary.